packages/composite-checkout/src/components/radio-button.js

```diff
@@ -100,11 +100,32 @@ const RadioButtonWrapper = styled.div`
 	:hover svg {
 		filter: grayscale( 0 );
 	}
+
+	${handleWrapperDisabled};
 `;
 
+function handleWrapperDisabled( { disabled } ) {
+	if ( ! disabled ) {
+		return null;
+	}
+
+	return `
+		:before,
+		:hover:before {
+			border: 1px solid lightgray;
+		}
+
+		svg,
+		:hover svg {
+			filter: grayscale( 100% );
+			opacity: 50%;
+		}
+	`;
+}
+
 const Radio = styled.input`
 	position: absolute;
-	opacity: 0;
+	opacity: 0 !important;
```

Comment:
I don't like using `!important` but had to in this case to override some very specific styles we were inheriting from Calypso.